modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -13,12 +13,187 @@
  */
 
 (function () {
+	const HEADING_BOTH = 'Both';
+	const HEADING_COL = 'Column';
+	const HEADING_NONE = 'None';
+	const HEADING_ROW = 'Row';
```

Comment:
[optional] To highlight these constants are in the same domain, have the same purpose, we can bundle them in an object:
```js
const HEADING = {
    BOTH: 'Both',
    ...
}
```